icpc-wf/ansible/roles/scoreboard/tasks/main.yml

```
@@ -0,0 +1,119 @@
+---
+# These tasks configure a static scoreboard
+
+- name: create static scoreboard directory
+  file:
+    path: /home/domjudge/scoreboard
+    state: directory
+    owner: domjudge
+    group: domjudge
+    mode: 0755
+
+- name: create directories for styling, scripts, fonts and flags
+  file:
+    path: /home/domjudge/scoreboard/{{ item }}
+    state: directory
+    owner: domjudge
+    group: domjudge
+    mode: 0755
+  with_items:
```

```suggestion
  loop:
```

The current doc has a preference for loop